gremlin-dotnet/test/Gremlin.Net.UnitTest/Driver/ConnectionTests.cs

```diff
@@ -165,6 +165,11 @@ public async Task ShouldHandleCloseMessageForInFlightRequestsAsync()
 
             // Release the connection close message.
             receiveSempahore.Release();
+            // delay to set new mocked response for mockedClientWebSocket
+            while (mockedClientWebSocket.Object.State != WebSocketState.CloseReceived)
```

Comment:
To avoid an infinite loop (in theory) - can we add a timeout, if the state hasn't been changed - test would fail with an error?

Reply:
added in https://github.com/Bit-Quill/tinkerpop/pull/9/commits/36078f63a5da765b90540239ca836b1398c5af57